<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qm2Cm</class>
 <widget class="QMainWindow" name="Qm2Cm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QMake2CMake</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>4</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>BaseDir:</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::NoTextInteraction</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_base"/>
      </item>
      <item>
       <widget class="QPushButton" name="pb_base">
        <property name="text">
         <string>select Dir</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="te_disp"/>
    </item>
    <item>
     <widget class="QPushButton" name="pb_start">
      <property name="text">
       <string>start conversion</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
